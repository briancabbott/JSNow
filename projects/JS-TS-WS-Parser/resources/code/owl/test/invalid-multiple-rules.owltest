---invalid = b
b = c
invalid = c
