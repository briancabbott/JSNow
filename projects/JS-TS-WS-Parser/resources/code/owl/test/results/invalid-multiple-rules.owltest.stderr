error: there are multiple rules named 'invalid'

  invalid = b
  ~~~~~~~    
  ...
  invalid = c
  ~~~~~~~    

